Mathlib/Order/SuccPred/Limit.lean

```diff
@@ -0,0 +1,418 @@
+/-
+Copyright (c) 2022 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+
+! This file was ported from Lean 3 source module order.succ_pred.limit
+! leanprover-community/mathlib commit 1e05171a5e8cf18d98d9cf7b207540acb044acae
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Order.SuccPred.Basic
+import Mathlib.Order.BoundedOrder
+
+/-!
+# Successor and predecessor limits
+
+We define the predicate `Order.isSuccLimit` for "successor limits", values that don't cover any
+others. They are so named since they can't be the successors of anything smaller. We define
+`Order.isSuccLimit` analogously, and prove basic results.
```

Comment:
```suggestion
We define the predicate `Order.IsSuccLimit` for "successor limits", values that don't cover any
others. They are so named since they can't be the successors of anything smaller. We define
`Order.IsPredLimit` analogously, and prove basic results.
```